js/search.js

```diff
@@ -949,25 +949,38 @@
 			for (var i = 0; i < moves.length; i++) {
 				var id = moves[i];
 				var isViable = BattleMovedex[id] && BattleMovedex[id].isViable;
-				if (id === 'aerialace') isViable = (toId(set.ability) === 'technician');
+				if (id === 'aerialace') isViable = (toId(set.species) in {scyther:1, aerodactylmega:1, kricketune:1});
 				if (id === 'bellydrum') isViable = (toId(set.species) in {azumarill:1, linoone:1, slurpuff:1});
-				if (id === 'counter') isViable = (toId(set.species) in {chansey:1, skarmory:1, clefable:1, wobbuffet:1});
+				if (id === 'counter') isViable = (toId(set.species) in {chansey:1, skarmory:1, clefable:1, wobbuffet:1, alakazam:1});
+				if (id === 'curse') isViable = (toId(set.species) === 'snorlax');
+				if (id === 'drainingkiss') isViable = (toId(set.ability) === 'triage');
 				if (id === 'dynamicpunch') isViable = (toId(set.ability) === 'noguard');
+				if (id === 'electroball') isViable = (toId(set.species) === 'raichualola');
+				if (id === 'endeavor') isViable = (toId(set.species) in {mamoswine:1, scolipede:1});
 				if (id === 'focuspunch') isViable = (toId(set.species) === 'breloom');
```

Comment:
No one else runs SubPunch?

Reply:
i mean i dont think even breloom 'runs' it anymore (not sure whether it did when somebody put it there) but its probably tolerable on a fair few others (megamawile? buzzwole? toxicroak?)

Reply:
Might as well just leave it generally Usable, it's a funny prediction move that could see niche use basically everywhere.